### DISCUSSION TITLE
How to access console logs of an installed app?

### DISCUSSION THREAD

**Discussion by elanzini** [tauri-apps/tauri#4803] [Q&A] [answered]
### Context:
I run`npm run tauri build`, get a `dmg` file and install my app locally.
I have Firebase integrated in my app and it works fine when I start it with `npm run tauri dev` but nothing opens up when clicking the login button in the installed app.

### Problem:
I am trying to access the `console` to see if the logs can point me in the right direction.
I tried right clicking as I would do in `dev` and `Inspect` but the option is not present in the menu.

How can I do that?

**Comment by FabianLars** [comment, marked as answer]
You can enable the devtools by adding the `"devtools"` feature flag to the `tauri` dependency in `Cargo.toml`.

Alternatively you can build a debug build via `npm run tauri build -- --debug` (output in `target/debug/bundle/`).

**Reply by elanzini** [reply]
Works perfectly! Thanks @FabianLars
